package.json

```diff
@@ -23,8 +23,8 @@
     "react-native": ">=0.60.0-rc.0 <1.0.x"
   },
   "devDependencies": {
-    "react": "16.13.1",
-    "react-native": "0.63.2"
+    "react": "17.0.1",
+    "react-native": "0.64.4"
```

Comment:
Any reason to not update this to "0.66.5" as well like the package json inside the example/ path?

Reply:
@alonshp I don't remember, did we had any issues with changing to the `0.66.5` and `17.0.2` like in the package json of the example project? Can you please take a look tmw?